Build.fsx

```diff
@@ -133,12 +134,28 @@ Target "TestOnly" (fun _ ->
     let parallelMode = if parallelizeTests then ParallelMode.All else ParallelMode.NoParallelization
     let maxThreads = if maxParallelThreads = 0 then CollectionConcurrencyMode.Default else CollectionConcurrencyMode.MaxThreads(maxParallelThreads)
 
+    let netCoreXunitProjects = [
+        "AutoFixtureUnitTest"
+        "AutoFixtureDocumentationTest"
+    ]
```

Comment:
I run these unit tests using `dotnet xunit` that is available only if particular packages are installed to the project. Currently only 2 ones are supported. Therefore, only selected projects are compatible ATM. I expect that eventually each test project will be compatible, so we could get rid of the list.